app/cdap/components/WrangleHome/Components/OngoingDataExplorationsCard/index.tsx

```diff
@@ -41,10 +41,15 @@ export default function OngoingDataExplorationCard({ item }) {
     >
       {item.map((eachItem, index) => {
         switch (eachItem.type) {
+          case 'icon':
+            return (
+              <Grid item xs={3} key={index} className={`${classes.connectorIcon}`}>
```

Comment:
we dont need string literals here
we can directly do className={classes.connectorIcon}

Reply:
Done.